src/ansible_builder/_target_scripts/introspect.py

```diff
@@ -1,16 +1,16 @@
 import argparse
-import importlib.metadata
 import logging
 import os
+import re
 import sys
 import yaml
 
-import requirements
-
-
 base_collections_path = '/usr/share/ansible/collections'
 logger = logging.getLogger(__name__)
 
+REQ_NORM_RE = re.compile(r'[-_.]+')
```

Comment:
https://peps.python.org/pep-0503/#normalized-names

Reply:
Thanks for the explanation. Was a head scratcher for me.